utils/forest_shim/src/sector.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+pub use fvm_shared::sector::RegisteredSealProof as RegisteredSealProofV2;
+pub use fvm_shared::sector::SectorInfo as SectorInfoV2;
+pub use fvm_shared3::sector::RegisteredSealProof as RegisteredSealProofV3;
+pub use fvm_shared3::sector::SectorInfo as SectorInfoV3;
```

Comment:
Could we not expose those? The point here is to abstract away the specific version.

Reply:
removed pub